examples/prometheus/prometheus.yaml

```diff
@@ -438,67 +551,40 @@ objects:
           - targets:
             - "localhost:9093"
 
-# Create a fully end-to-end TLS connection to the alert proxy
-- apiVersion: route.openshift.io/v1
-  kind: Route
-  metadata:
-    name: alerts
-    namespace: "${NAMESPACE}"
-  spec:
-    to:
-      name: alerts
-    tls:
-      termination: Reencrypt
-      insecureEdgeTerminationPolicy: Redirect
-- apiVersion: v1
-  kind: Service
-  metadata:
-    annotations:
-      service.alpha.openshift.io/serving-cert-secret-name: prometheus-alerts-tls
-    labels:
-      name: alerts
-    name: alerts
-    namespace: "${NAMESPACE}"
-  spec:
-    ports:
-    - name: alerts
-      port: 443
-      protocol: TCP
-      targetPort: 9443
-    selector:
-      app: prometheus
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: alerts-proxy
-    namespace: "${NAMESPACE}"
-  stringData:
-    session_secret: "${SESSION_SECRET}="
-
 - apiVersion: v1
   kind: ConfigMap
   metadata:
-    name: prometheus-alerts
+    name: prometheus-alertmanager
     namespace: "${NAMESPACE}"
   data:
     alertmanager.yml: |
       global:
-
+        #smtp mail configuration for mail alerts
+        smtp_smarthost: 'mail.exammple.com:25'
+        smtp_from: 'openshift@cluster.local'
+        smtp_auth_username: 'admin'
+        smtp_auth_password: 'password'
       # The root route on which each incoming alert enters.
       route:
         # default route if none match
-        receiver: alert-buffer-wh
-
+        receiver: all
         # The labels by which incoming alerts are grouped together. For example,
         # multiple alerts coming in for cluster=A and alertname=LatencyHigh would
         # be batched into a single group.
         # TODO:
         group_by: []
-
         # All the above attributes are inherited by all child routes and can
         # overwritten on each.
-
       receivers:
       - name: alert-buffer-wh
         webhook_configs:
         - url: http://localhost:9099/topics/alerts
+      - name: mail
```

Comment:
Remove, this won't work out of the box.

Reply:
am. It did when I tested. I can retest and confirm. Might did it wrong somewhere.

Reply:
Ou. Got you, sr. Will remov